srcMain/main.cpp

```diff
@@ -124,7 +124,8 @@ void shasta::main::main(int argumentCount, const char** arguments)
 
 
     // Execute the requested command.
-    if(assemblerOptions.commandLineOnlyOptions.command == "assemble") {
+    if(assemblerOptions.commandLineOnlyOptions.command == "assemble" or
```

Comment:
Please update the `throw` message about 20 lines below to also include `filterReads` as one of the valid commands.

Reply:
And similarly add the new command to `createBashCompletionScript`.